Cargo.toml

```diff
@@ -13,14 +13,14 @@ chrono = "0.4.26"
 clokwerk = "0.4.0"
 futures = "0.3.28"
 fallible-iterator = "0.2.0"
-influxdb2-structmap = { git = "https://github.com/foundation-zero/influxdb2.git" }
+influxdb2-structmap = "0.2.0"
 itertools = "0.11.0"
 tokio = { version = "1.29.1", features = ["rt", "rt-multi-thread", "macros", "io-util"], default-features = false }
 tokio-stream = "0.1.14"
 thiserror = "1.0.45"
 clap = { version = "4.3.21", features = ["derive", "env"] }
 env_logger = "0.10.0"
 log = "0.4.20"
-influxdb2 = { git = "https://github.com/foundation-zero/influxdb2.git", features = ["rustls", "gzip"], default-features = false }
+influxdb2 = { version = "0.5.0", features = ["rustls", "gzip"], default-features = false }
```

Comment:
Switching to the main influxdb2 crate, that now has support for any reqwest client builder